### DISCUSSION TITLE
Java9 and AbstractTensor.java

### DISCUSSION THREAD

**Discussion by carlosuc3m** [bytedeco/javacpp-presets#1129] [Q&A] [answered]
HEllo, 
I am creating the presets for different Pytorch versions and I found two files that I don't really know if I have to modify for different versions or if they are going to be created by Javacpp. Those files are:
https://github.com/bytedeco/javacpp-presets/blob/master/pytorch/src/main/java/org/bytedeco/pytorch/AbstractTensor.java
https://github.com/bytedeco/javacpp-presets/blob/master/pytorch/src/main/java9/module-info.java
and I am asking it because they are located in a different place with respect to the others.
REgards,
Carlos

**Comment by saudet** [comment, marked as answer]
Like I keep telling you, different versions have different features. That's what versions are for! I don't think you'll need to modify those files in particular, but it's always possible that you might.
